modules/router/templates/assets_origin.conf.erb

```diff
@@ -40,6 +40,9 @@ server {
 
   <%- @asset_routes.each do |alias_path, vhost_name| -%>
   location <%= alias_path %> {
+    <%- if alias_path =~ %r(/government/uploads/) && vhost_name =~ %r(whitehall-frontend) -%>
```

Comment:
Minor: Is there a reason why you used regular expression comparisons in this condition rather than equality? As far as I can tell, you only want this to match for a single entry in the `router::assets_origin::asset_routes` hash. Or am I missing something?

Reply:
You're quite right. I've added another commit to convert this to use string comparison which hopefully makes it clearer that I'm only expecting the header to be added to a single entry in the `router::assets_origin::asset_routes` hash.